moment_kinetics/src/external_sources.jl

```diff
@@ -949,22 +1032,23 @@ Add external source term to the ion kinetic equation.
             end
         else
             error("Unsupported combination in source_type=$(source_type) evolve_density=$(moments.evolve_density), "
-                  * "evolve_upar=$(moments.evolve_upar), evolve_ppar=$(moments.evolve_ppar)")
+                  * "evolve_upar=$(moments.evolve_upar), evolve_p=$(moments.evolve_p)")
         end
     elseif source_type == "beam" || source_type == "beam-with-losses"
         @begin_s_r_z_region()
         source_vpa0 = ion_source.source_vpa0
         source_vperp0 = ion_source.source_vperp0
+        Maxwellian_prefactor = 1.0 / π^1.5
         if !(source_vpa0 > 1.0e-8)
```

Comment:
Why is this check here? Surely vpa0 can be negative?

Reply:
Whoops, I should have checked further. I don't recall what is going on here, clearly.